pkgs/memory_usage/pubspec.yaml

```
@@ -0,0 +1,16 @@
+name: memory_usage
+version: 0.1.0
```

I'd like to follow up this PR w/ one or two changes (if I don't end up doing that in the next day or two we can remove this wip and publish).

```suggestion
version: 0.1.0-wip
```